lib/rubocop/cop/metrics/block_length.rb

```diff
@@ -44,6 +44,10 @@ def excluded_methods
         def cop_label
           LABEL
         end
+
+        def_node_matcher :struct_constructor?, <<~PATTERN
+          (block (send (const nil? :Struct) :new ...) ...)
+        PATTERN
```

Comment:
This could be moved `as is` to `rubocop-ast` or can be merged with `class_constructor?`. The latter will impact other places where `class_constructor?` is used.

Reply:
I'd rather keep them separate. It's a good idea to move this to `rubocop-ast` eventually.

Reply:
I opened https://github.com/rubocop-hq/rubocop-ast/issues/28